core/src/main/resources/jenkins/install/pluginSetupWizard.properties

```diff
@@ -65,7 +65,7 @@ installWizard_installIncomplete_banner=Resume Installation
 installWizard_installIncomplete_message=Jenkins was restarted during installation and some plugins did not seem to get installed.
 installWizard_installIncomplete_resumeInstallationButtonLabel=Resume
 installWizard_saveFirstUser=Save and Continue
-installWizard_skipFirstUser=Continue as admin
+installWizard_skipFirstUser=Skip form and continue as admin
```

Comment:
Unsure, but perhaps?

```suggestion
installWizard_skipFirstUser=Skip and continue as admin
```

Reply:
I agree with the need to trim it.

The other option to be more concise could be simply _skip_ as there is clarification at the end of the setup wizard:
![postUserCreateFormClickedContinueAsAdmin](https://user-images.githubusercontent.com/46274278/79904152-40b9fe00-83e2-11ea-81bc-ae050c5c78f8.png)